zerver/lib/url_preview/preview.py

```diff
@@ -24,8 +23,12 @@
     r'(?::\d+)?'  # optional port
     r'(?:/?|[/?]\S+)$', re.IGNORECASE)
 # FIXME: This header and timeout are not used by pyoembed, when trying to autodiscover!
-# Set a custom user agent, since some sites block us with the default requests header
-HEADERS = {'User-Agent': 'Zulip URL preview/%s' % (ZULIP_VERSION,)}
+# Use Chrome User-Agent, since some sites refuse to work on old browsers
+CHROME_USER_AGENT = (
+    'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) '
+    'Chrome/81.0.4044.113 Safari/537.36'
+)
+HEADERS = {'User-Agent': CHROME_USER_AGENT}
```

Comment:
Is there a way we can add some extra content that at least disambiguates that we're a bot rather than looking like Chrome so that we're not being bad citizens here?  E.g. putting something before or after their data?  

@andersk FYI in case you have ideas.

Reply:
I think we can change the OS part of the string. I'll send a new PR after some testing.

Reply:
@timabbott updated this PR to add some information about Zulip to the User Agent string.

Reply:
I would suggest adopting exactly the same convention that [Googlebot uses](https://support.google.com/webmasters/answer/1061943?hl=en), which is only slightly different:

`Mozilla/5.0 AppleWebKit/537.36 (KHTML, like Gecko; compatible; ZulipURLPreview/{}; +https://zulipchat.com) Chrome/81.0.4044.113 Safari/537.36`

I’ve tested this against music.youtube.com.

Reply:
Thanks @andersk !

Reply:
I think we want the host of the server, not zulipchat.com?  I pushed back here with that change.

Reply:
From the way I see it, this URL is for webmasters to know more about the program that is hitting their server. To know where this is running, they can always look at the IP address in their logs. So, I think zulipchat.com may be more appropriate here.

Reply:
Well, Googling for `ZulipURLPreview` will already get you to "Zulip" immediately, so I'm not sure if it adds anything to add a zulipchat.com here?  Might be better to not have a URL here and just plan to add one later when we have a Help Center page we want to direct folks interested in this to.

Reply:
That sounds reasonable to me!